elastic/apm-server
# [7.x] New tracer (#3836)

## Motivation/summary

* Backports https://github.com/elastic/apm-server/pull/3836\

* Also includes https://github.com/elastic/apm-server/pull/3909/commits/4bed7bb64eb05feed6aa70dad37e78c68b7aeb30 and a Beats update

## Checklist


I have considered changes for:

## How to test these changes

- Try both the new configuration and old instrumentation config settings and see that they both work, but the new one takes precedence.
- See transactions from both libbeat and apm-server in the UI (or Elasticsearch index if you prefer)
- Test that is disabled by default
- Test it with explicitly setting `instrumentation.enabled=false`, and see no transactions in the UI.
